packages/nextjs-kit/src/components/contentWithImage.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Image from 'next/image';
+import type { ImageProps } from 'next/image';
+import { useRouter } from 'next/router';
+
+interface ContentProps {
+	title: string;
+	imageProps?: ImageProps;
+	content: string;
+	date?: string;
+}
+
+/**
+ *
+ * @param props - The props needed for the ContentWithImage component
+ * @param props.title - The title of your content
+ * @param props.imageProps - All props the user wishes to pass to the next/image component
+ * @remarks
+ * imageProps is an optional prop to be used if there is an image to be associated with the content.
+ * If imageProps is used it is required that the user passes in values for src, width, and height
+ * See the documentation link below for more information on optional and required props
+ * @see {@link https://nextjs.org/docs/api-reference/next/image} for all next/image documentation
+ * @param props.content - A string of elements to make up the content on the post
+ * @param props.date - An optional date to be displayed on the post
+ * @param props.previousPagePath - The path of the previous page to navigate back to
+ * @returns A component with a featured image and content passed by the user
+ */
+const ContentWithImage: React.FC<ContentProps> = ({
+	title,
+	imageProps,
+	content,
+	date,
+}: ContentProps): JSX.Element => {
+	const router = useRouter();
+	return (
+		<article className="prose lg:prose-xl mt-10 mx-auto">
+			<h1>{title}</h1>
+			{date ? <p className="text-sm text-gray-600">{date}</p> : null}
+
+			<a onClick={() => router.back()} className="font-normal">
+				Back &rarr;
+			</a>
+
+			<div className="mt-12 max-w-lg mx-auto lg:grid-cols-3 lg:max-w-screen-lg">
+				{imageProps ? (
+					<div
+						className="relative w-full rounded-lg shadow-lg overflow-hidden mb-10"
+						style={{ height: '50vh' }}
```

Comment:
This is another artifact from the quick'n'dirty implementation of this in the starters. When I give the imageProps a height and width, I would expect it to be respected by this container as well.

Code:
```jsx
<ContentWithImage
    title={title}
    content={body.processed}
    imageProps={{ src: imgSrc, width: 200, height: 100 }}
/>
```

Result:
![image](https://user-images.githubusercontent.com/63015754/190446972-5d9837ca-1b59-4018-8638-ea3647a34d33.png)